libminifi/include/FlowController.h

```diff
@@ -87,13 +91,8 @@ class FlowController : public core::controller::ForwardingControllerServiceProvi
     return this->provenance_repo_;
   }
 
-  // Get the flowfile repository
-  virtual std::shared_ptr<core::Repository> getFlowFileRepository() {
-    return this->flow_file_repo_;
-  }
-
   // Load flow xml from disk, after that, create the root process group and its children, initialize the flows
-  virtual void load(const std::shared_ptr<core::ProcessGroup> &root = nullptr, bool reload = false);
+  virtual void load(std::unique_ptr<core::ProcessGroup>&& root = nullptr, bool reload = false);
```

Comment:
I don't know what do you mean by "the implementation will have to move out from the parameter", because it's already have to be moved out. Anyways, I changed the parameter to be by-value.

Reply:
Did you forget to push the changes?

Reply:
Okay, this makes sense. I will use by value more from now on.
Changes will be pushed soon, I'm building and testing them.